ParquetClassLibrary/Characters/BeingStatus.cs

```diff
@@ -0,0 +1,49 @@
+using Newtonsoft.Json;
+using ParquetClassLibrary.Sandbox;
+
+namespace ParquetClassLibrary.Characters
+{
+    public class BeingStatus
+    {
+        /// <summary>The <see cref="Being"/> whose status is being tracked.</summary>
+        [JsonProperty(PropertyName = "in_beingDefinition")]
+        public Being BeingDefinition { get; }
+
+        /// <summary>The <see cref="Location"/> the tracked <see cref="Being"/> occupies.</summary>
+        [JsonProperty(PropertyName = "in_currentLocation")]
+        public Location CurrentLocation { get; set; }
```

Comment:
nit: Current seems like an odd prefix for things. These are different than Parquets, where there are instances of objects, but saying that "Everything defined at the moment is defined at the moment" may be redundant.

That said, i definitely prefix things with cur, and this does avoid the awkwardness of "public Location Location". This nit is to say, this isn't a perfect solution, but it is awfully similar to what I already do.

Reply:
Fair.  I'd been saying "position" elsewhere, not sure why I changed it.  Updating.